stdlib/Statistics/src/Statistics.jl

```diff
@@ -73,7 +73,29 @@ function mean(f, itr)
     end
     return total/count
 end
-mean(f, A::AbstractArray) = sum(f, A) / length(A)
+
+"""
+    mean(f::Function, A::AbstractArray, [dims])
```

Comment:
```suggestion
    mean(f::Function, A::AbstractArray; dims)
```
(just like for `mean(f, A)` below)